### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by mikesaelim**
* Bump the ruby version used in development to 3.0.0
* Add ruby 2.7.2 and 3.0.0 to the Travis build matrix
* Upgrade rubocop to the latest version, v1.9.1, and fix rubocop violations
* Configure rspec to emit warnings
* Fix a broken link in the README

Since this PR contains no functional changes to the gem, I'm not planning on cutting a release after merging.

### CODE REVIEW

File: ruby_audit.gemspec [hunk 1]
```diff
@@ -15,8 +15,9 @@ Gem::Specification.new do |spec|
                        'version. It complements bundler-audit, providing ' \
                        'complete coverage for your Ruby stack.'
   spec.homepage      = 'https://github.com/civisanalytics/ruby_audit'
-  spec.license       = 'GPLv3'
+  spec.license       = 'GPL-3.0-or-later'
```
Comment (aripollak) [inline ruby_audit.gemspec:18]: Looking around at the license declarations in this repo, I think we also need to include the full GPL 3.0 text in LICENSE, and change the README to say this is licensed under GPL 3.0 or later. That could be a followup PR though.

File: .rubocop.yml [hunk 2]
```diff
@@ -1,12 +1,17 @@
-Metrics/LineLength:
+AllCops:
+  TargetRubyVersion: 2.5
+  NewCops: disable
```
Comment (aripollak) [inline .rubocop.yml:3]: Will we be re-enabling these later? They seem useful.

**Review by aripollak — APPROVED** [review]
LGTM

File: .rubocop.yml (hunk 2 above)
Comment (mikesaelim) [reply .rubocop.yml:3]: I wasn't enthusiastic about either specifying `NewCops: enable` and fixing things every Rubocop upgrade, or individually enabling all the current cops.  But maybe the former isn't so bad?

Comment (aripollak) [reply .rubocop.yml:3]: My reading of the new option is that if you specify neither, it won't enable the new cops but will warn you that there are new cops to explicitly enable/disable?

Comment (aripollak) [reply .rubocop.yml:3]: I'd support either one though, so we eventually get improvements from new cops.

Comment (mikesaelim) [reply .rubocop.yml]: The long list of warnings from specifying neither was pretty annoying, so I went with specifying `enable`.

**Review by aripollak — APPROVED** [review]
LGTM

**Pull request merged**